app/overrides/spree/admin/orders_controller_override.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  module Admin
+    module OrdersControllerOverride
```

Comment:
Curious why we chose this path over adding a `TransactionSyncLogController` with an `index` action for this 🤔 In general I prefer we limit the amount of overriding and in this case it doesn't seem necessary.

Reply:
Mostly because it seemed like a nice touch to make it a tab on the orders page, instead of it's own page! You are right though comes at a cost to maintainability in the long run. Could go either way on this one!

Reply:
> Mostly because it seemed like a nice touch to make it a tab on the orders page
I don't think this prevents us from using a separate controller. I don't think this action really belongs on the orders controller, and I think it would be more appropriate as a nested resource on orders.

I am also thinking we may eventually add a `#show` action if we need to display the full details of a log entry.

Reply:
Resolved IRL!